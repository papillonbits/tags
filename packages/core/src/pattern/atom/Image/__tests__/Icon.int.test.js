import { cross, check } from './Icon.int.story'

describe('<Icon />', () => {
  describe('Render', () => {
    test('must match cross', () => {
      expect(global.renderToJSON(cross())).toMatchSnapshot()
    })

    test('must match check', () => {
      expect(global.renderToJSON(check())).toMatchSnapshot()
    })
  })
})
