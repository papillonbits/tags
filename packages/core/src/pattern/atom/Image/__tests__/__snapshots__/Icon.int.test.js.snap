// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> Render must match check 1`] = `
<div>
  <svg
    font-size="1px"
    height="10"
    viewBox="0 0 1024 1024"
    width="10"
  >
    <path
      d="M896 213.333q18.333 0 30.5 12.167t12.167 30.5q0 18-12.333 30.333l-512 512q-12.333 12.333-30.333 12.333t-30.333-12.333l-256-256q-12.333-12.333-12.333-30.333 0-18.333 12.167-30.5t30.5-12.167q18 0 30.333 12.333l225.667 226 481.667-482q12.333-12.333 30.333-12.333z"
    />
  </svg>
</div>
`;

exports[`<Icon /> Render must match cross 1`] = `
<div>
  <svg
    font-size="1px"
    height="10"
    viewBox="0 0 1024 1024"
    width="10"
  >
    <path
      d="M810.667 170.667q18.333 0 30.5 12.167t12.167 30.5q0 18-12.333 30.333l-268.667 268.333 268.667 268.333q12.333 12.333 12.333 30.333 0 18.333-12.167 30.5t-30.5 12.167q-18 0-30.333-12.333l-268.333-268.667-268.333 268.667q-12.333 12.333-30.333 12.333-18.333 0-30.5-12.167t-12.167-30.5q0-18 12.333-30.333l268.667-268.333-268.667-268.333q-12.333-12.333-12.333-30.333 0-18.333 12.167-30.5t30.5-12.167q18 0 30.333 12.333l268.333 268.667 268.333-268.667q12.333-12.333 30.333-12.333z"
    />
  </svg>
</div>
`;
